apps/game_backend/priv/daily_rewards_rules.json

```diff
@@ -0,0 +1,11 @@
+{
+    "reward_per_day": {
+        "1": "2",
```

Comment:
Is this supposed to be `day: next_day` or `day: reward_for_day`?

Reply:
Since its hard to follow, I reworked this config to be more descriptive.